lib/Crypto/doc/sailfish-crypto-overview.qdoc

```diff
@@ -24,7 +24,7 @@ Sailfish OS Secrets and Crypto Framework.
 \note The Sailfish OS Secrets and Crypto Framework is still in active
 development and is subject to change.  Normal 3rd party API backward
 compatibility promises don't hold.
-Follow \l{http://together.jolla.com/}{Together Jolla} release notes and
+Follow \l{https://forum.sailfishos.org/}{Sailfish OS Forum} release notes and
 \l{https://lists.sailfishos.org/cgi-bin/mailman/listinfo/devel}
```

Comment:
The existing changes already look very sensible to me. It would make sense to remove the references to the mailing list too, given these are no longer active. So, just "Follow Sailfish OS Forum release notes to get notified of API changes."